tests/spec/core/webidl-spec.js

```diff
@@ -707,6 +707,17 @@ partial dictionary AnotherThing {
     expect(idl.querySelector(".idlSectionComment")).toBeNull();
   });
 
+  it("uniquely links to enum values", () => {
+    const target = doc.getElementById("multipleEnums");
+    const idlLinks = target.querySelectorAll("a[data-link-for]");
```

Comment:
Nit:
can probably just do `doc.querySelectorAll("#multipleEnums a[data-link-for]");`